src/app/ecosystem-explorer/project-form/schema/EcosystemProjectFormSchema.ts

```diff
@@ -58,11 +58,10 @@ export const EcosystemProjectFormSchema = z.object({
       message: 'The description is too long',
     }),
   category: OptionSchema,
-  topic: OptionSchema,
-  files: z
-    .array(z.instanceof(File))
-    .refine(validateOneFileSelected, { message: 'A logo is required' })
-    .refine(validateFileSizes, { message: 'Logo size exceeds the limit' }),
+  subcategory: OptionSchema,
+  logo: z
+    .instanceof(File, { message: 'A logo is required' })
+    .refine(validateFileSize, { message: 'Logo size exceeds the limit' }),
```

Comment:
Which limit?

Reply:
Let' use `MAX_FILE_SIZE_IN_BYTES` here and also here "For best quality, please submit a white logo with a transparent background, at least 1000px by 1000px, and under 100KB. You can use tools like"

Reply:
✅